examples/gameroom/daemon/src/authorization_handler/mod.rs

```diff
@@ -68,14 +68,25 @@ pub fn run(
         },
     );
 
+    ws.set_reconnect(true);
+    ws.set_reconnect_limit(10);
```

Comment:
Should these be using a configured value (as in the previous commit configures)?

Reply:
Do you mean just using the defaults establish in the server side?

Reply:
addressed